apps/web/src/services/asset-inventory/collector/type.ts

```diff
@@ -27,6 +27,12 @@ export const CollectorQueryHelperSet = {
     STATE: 'state',
 } as const;
 
+export const CollectorItemInfoType = {
```

Comment:
nitpick: For literal types, it's recommended to use UPPER_SNAKE_CASE :)

```suggestion
export const COLLECTOR_ITEM_INFO_TYPE = {
```